Flank/src/main/java/com/walmart/otto/configurator/Configurator.java

```diff
@@ -201,6 +204,22 @@ public void setDisableAutoGoogleLogin(boolean disableAutoGoogleLogin) {
     this.disableAutoGoogleLogin = disableAutoGoogleLogin;
   }
 
+  public boolean recordVideo() {
+    return recordVideo;
+  }
+
+  public void setRecordVideo(boolean shouldRecordVideo) {
+    recordVideo = shouldRecordVideo;
```

Comment:
the pattern seems to be:

`this.recordVideo = recordVideo`

It'd be nice to keep the code style consistent.